ios/RCCNavigationController.m

```diff
@@ -211,6 +211,8 @@ - (void)performAction:(NSString*)performAction actionParams:(NSDictionary*)actio
     
     RCCViewController *viewController = [[RCCViewController alloc] initWithComponent:component passProps:passProps navigatorStyle:navigatorStyle globalProps:nil bridge:bridge];
     
+    viewController.navigationItem.hidesBackButton = YES;
```

Comment:
> I was under the assumption that it was normal on iOS side to have to specify hiding the back button when performing any routing.

I don't think that's necessarily true. I think navigation stacks should always be constructed in a way that is useful for the user to navigate using the back button. I would say hiding the back button is almost a bit unnatural, in terms of iOS interface guidelines.

From the [iOS Human Interface Guidelines](https://developer.apple.com/ios/human-interface-guidelines/interaction/navigation/):
> *Use a navigation bar to traverse a hierarchy of data*. The navigation bar’s title can show the current position in the hierarchy, and the back button makes it easy to return to the previous location.